lib/copy/__tests__/copy-broken-symlink.test.js

```diff
@@ -5,7 +5,7 @@ const os = require('os')
 const fse = require('../..')
 const path = require('path')
 const assert = require('assert')
-const copy = require('../copy')
+const { copy } = require('../')
```

Comment:
I need to change the import here. `lib/copy/copy.js` now exports the promise-only API (instead of the callback-only API), while `lib/copy/index.js` now exports the unified API.